lib/Util/Aggregation.php

```diff
@@ -127,7 +127,7 @@ public static function getAggregationDateFormat($level) {
 	 * @param  string  $targetLevel desired highest level (e.g. 'day')
 	 * @return array|boolean list of valid aggregation levels
 	 */
-	public function getOptimalAggregationLevel($uuid, $targetLevel = null) {
+	public function hasDataForAggregationLevel($uuid, $targetLevel = null, $from = null, $to = null) {
```

Comment:
The `for` loop at the end of the function should no longer be needed as we already know what level we're testing against. Shouldn't we now simplify the return type?

Reply:
This change would really break the test.